InteropDotNet/LibraryLoader.cs

```diff
@@ -45,21 +45,65 @@ public IntPtr LoadLibrary(string fileName, string platformName = null)
             }
         }
 
+        /// <summary>
+        /// This is a utility to override environments where the LoadLibrary resolution fails. This happens when an executing DLL is deleted (temp file) or a DLL is loaded from a stream.
+        /// </summary>
+        /// <param name="dllFile">The full path to the DLL file.</param>
+        /// <returns>A pointer to the loaded DLL or IntPtr.Zero if unable to load.</returns>
+        public IntPtr LoadLibrary(string dllFile)

```

Comment:
Could you please clarify use-cases when this method should be actually executed (Currently, `InteropRuntimeImplementer` always use the overload of `LoadLibrary` with two arguments). If you suppose that this method should be called manually by a user, this API looks very confusing because we already have an overload of `LoadLibrary` with two arguments where the second argument is optional (users will hit problems with method resolution because the new overload hides the previous one; it's a pretty strange breaking change). Also, it's not obvious how to choose the appropriate API in the manual call case.